ui/node/attributes.go

```diff
@@ -98,6 +98,8 @@ type InputAttributes struct {
 	//
 	// required: true
 	NodeType string `json:"node_type"`
+
+	DataWebauthnIdentifier bool `json:"-" faker:"-"`
```

Comment:
Carries state that is used in another context. Leaky?

Reply:
fixed